### ISSUE TITLE
Closure messages in validation not work correctly

### ISSUE THREAD

**Issue by yshelomentsev** [poppinss/indicative#251]
Hi, indicative has feature like closures in messages, but it fails if closures count is more than 1.

Example of **correct** work (code throws _"Field «lead_id» must be filled"_):
```
const rules = {
  lead_id: 'required|integer',
  date: 'required|date'
}

const messages = {
  required: `Field «{{field}}» must be filled`
}

await validate({}, rules, messages)
```

Example of **bug** (code throws default _"required validation failed on lead_id"_ instead of _"Field «lead_id» must be filled"_):
```
const rules = {
  lead_id: 'required|integer',
  date: 'required|date'
}

const messages = {
  required: `Field «{{field}}» must be filled`,
  integer: `Field «{{field}}» must be number`,
  date: `Field «{{field}}» must be date`
}

await validate({}, rules, messages)
```

It bugs and with closure and with micro templating.

**Comment by thetutlage** [comment]
Yup looks like a bug. I’ll look into it

**Comment by thetutlage** [comment]
Fixed in `7.2.1`
